packs/autoscale/actions/workflows/asg_deflate.yaml

```diff
@@ -16,83 +16,82 @@ workflows:
       get_chatops_room:
         action: st2.kv.get
         input:
-          key: 'asg.{$.asg}.channel'
+          key: 'asg.<% $.asg %>.channel'
         publish:
-          channel: '{$.get_chatops_room.result}'
+          channel: '<% $.get_chatops_room.result %>'
         on-success:
           - get_current_epoch
       get_current_epoch:
         action: st2.epoch
         publish:
-          current_epoch: $.get_current_epoch.result
+          current_epoch: <% $.get_current_epoch.result %>
         on-success:
           - get_last_deflate_timestamp
       get_last_deflate_timestamp:
         action: st2.kv.get
         input:
-          key: 'asg.{$.asg}.last_deflate_timestamp'
+          key: 'asg.<% $.asg %>.last_deflate_timestamp'
         publish:
-          last_deflate_timestamp: int($.get_last_deflate_timestamp.result)
+          last_deflate_timestamp: <% int($.get_last_deflate_timestamp.result) %>
         on-success:
           - get_deflate_delay
       get_deflate_delay:
         action: st2.kv.get
         input:
-          key: 'asg.{$.asg}.deflate_delay'
+          key: 'asg.<% $.asg %>.deflate_delay'
         publish:
-          deflate_delay: int($.get_deflate_delay.result)
+          deflate_delay: <% int($.get_deflate_delay.result) %>
```

Comment:
This I need to test. I recall running into an issue with the K/V store and types, which is why the cast is here. This is certainly one thing that needs testing tomorrow. 

-James

> On Feb 26, 2015, at 11:16 PM, Dmitri Zimin(e) <notifications@github.com> wrote:
> 
> In packs/autoscale/actions/workflows/asg_deflate.yaml:
> 
> >          publish:
> > -          deflate_delay: int($.get_deflate_delay.result)
> > +          deflate_delay: <% int($.get_deflate_delay.result) %>
> I wonder if int() is still needed here...
> 
> —
> Reply to this email directly or view it on GitHub.
>